build.sh

```diff
@@ -11,8 +11,8 @@ cd ../
 
 yarn generate
 
-mv .output/public/api/_content _site/api
+rsync -a .output/public/api/_content _site/api
```

Comment:
We can also add a `--remove-source-files` if we want to remove the files from the source.
Does it matter? What do we think? 🤨